test/extensions/filters/http/cache/http_cache_utils_test.cc

```diff
@@ -75,6 +75,37 @@ TEST_P(EffectiveMaxAgeTest, EffectiveMaxAgeTest) {
             std::chrono::seconds(GetParam().effective_max_age_secs));
 }
 
+void testReadAndRemoveLeadingDigits(absl::string_view s, int64_t expected,
+                                    absl::string_view remaining) {
+  absl::string_view input(s);
+  auto output = HttpCacheUtils::readAndRemoveLeadingDigits(input);
+  if (output) {
+    EXPECT_EQ(output, static_cast<uint64_t>(expected));
```

Comment:
suggest annotating your EXPECT_EQ calls with the input, so if it fails you can readily find which case it is.
e.g.
```
EXPECT_EQ(output, static_cast<uint64_t>(expected)) << "input=" << input;
```

Note that this is not generally needed, but because you are doing your EXPECTs in a helper method called multiple places, you will not be able to tell from a test log which test fails, as it will only print the line number of the EXPECT and not the line-number a stack-frame up.